cli/azd/pkg/osutil/expandable_string.go

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package project
+package osutil
```

Comment:
Moving `ExpandableString` out of `project` package and into our more generic `osutil` to help avoid circular dependencies between packages.